client/features/navbar/Navbar.js

```diff
@@ -14,24 +15,38 @@ const Navbar = () => {
 
   return (
     <div>
-      <h1>FS-App-Template</h1>
-      <nav>
-        {isLoggedIn ? (
-          <div>
-            {/* The navbar will show these links after you log in */}
-            <Link to="/home">Home</Link>
-            <Link to="/products">Shop</Link>
-            <button type="button" onClick={logoutAndRedirectHome}>
-              Logout
-            </button>
+      <nav className="relative container mx-auto px-8 bg-[#f5f4f0]">
+        {/* Flex container */}
+        <div className="flex items-center justify-evenly">
+          {/* Logo - was previously image, using text for simplicity for now */}
+          <div className="text-3xl font-bungee pl-1 pr-3 text-tahiti">
+            Placeholder
           </div>
-        ) : (
-          <div>
-            {/* The navbar will show these links before you log in */}
-            <Link to="/login">Login</Link>
-            <Link to="/signup">Sign Up</Link>
-          </div>
-        )}
+          {isLoggedIn ? (
+            <div className="hidden md:flex space-x-6 text-sm uppercase font-plex tracking-widest">
+              {/* The navbar will show these links after you log in */}
+              <Link to="/home" className="hover:text-tahiti">
+                Home
+              </Link>
+              <Link to="/products" className="hover:text-tahiti">
+                Shop
+              </Link>
+              <button
+                type="button"
+                onClick={logoutAndRedirectHome}
+                className="hover:text-tahiti"
+              >
+                LOGOUT
+              </button>
+            </div>
+          ) : (
+            <div className="hidden md:flex space-x-6 text-sm uppercase font-plex tracking-widest">
+              {/* The navbar will show these links before you log in */}
+              <Link to="/login">Login</Link>
+              <Link to="/signup">Sign Up</Link>
```

Comment:
will want our NavBar to have /home & /products even if they're a guest; maybe same as LoggedIn except the logout changes to login & register buttons

Reply:
Thank you! Fixed in this next push. Do we want /home and /products to be viewable by guests? Right now that's not possible (see AppRoutes.js) but I believe it's a simple change

Reply:
At present I made the HOME and PRODUCTS buttons on the NavBar visible, but click on them and you'll continue to be prompted for a login

Reply:
yes i think we should change to visible